kaldb/src/test/java/com/slack/kaldb/writer/LogMessageWriterImplTest.java

```diff
@@ -162,14 +162,7 @@ public void testApiLogMessageInsertion() throws Exception {
     final String testIndex = "hhvm_api_log";
     assertThat(searchChunkManager(testIndex, "").hits.size()).isEqualTo(1);
     assertThat(searchChunkManager(testIndex, "http_method:POST").hits.size()).isEqualTo(1);
-    assertThat(searchChunkManager(testIndex, "http_method:GET").hits.size()).isEqualTo(0);
-    assertThat(searchChunkManager(testIndex, "method:callbacks*").hits.size()).isEqualTo(1);
-    assertThat(searchChunkManager(testIndex, "http_method:POST AND method:callbacks*").hits.size())
-        .isEqualTo(1);
-    assertThat(searchChunkManager(testIndex, "http_method:GET AND method:callbacks*").hits.size())
-        .isEqualTo(0);
-    assertThat(searchChunkManager(testIndex, "http_method:GET OR method:callbacks*").hits.size())
-        .isEqualTo(1);
+    assertThat(searchChunkManager(testIndex, "http_method:GET").hits.size()).isEqualTo(0);  
```

Comment:
we're not really testing search behaviour here. None of these will ever fail so removing some . Just makes it tough to modify any data